src/structures/MessageMentions.js

```diff
@@ -134,7 +134,7 @@ class MessageMentions {
   }
 
   /**
-   * Any members that were mentioned (only in {@link TextChannel}s)
+   * Any members that were mentioned (only in {@link TextChannel}s and {@link ThreadChannel}s)
```

Comment:
It looks like this really only works in guilds. Perhaps it would be better to just say that it only works in guilds instead of adding every channel type in a guild that is text-based?

Reply:
Yes it could be better 👍